patches/minecraft/net/minecraft/client/renderer/ActiveRenderInfo.java.patch

```diff
@@ -1,5 +1,25 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/ActiveRenderInfo.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/ActiveRenderInfo.java
+@@ -28,7 +28,7 @@
+     private static float field_74587_g;
+     private static float field_74596_h;
+ 
+-    public static void func_74583_a(EntityPlayer p_74583_0_, boolean p_74583_1_)
++    public static void updateRenderInfo(Entity entityIn, boolean p_74583_1_)
```

Comment:
This needs to be an overloaded method for binary compatability.

Reply:
Talking about something like that?
```java
public static void func_74583_a(EntityPlayer p_74583_0_, boolean p_74583_1_)
{
    updateRenderInfo(p_74583_0_, p_74583_1_);
}
public static void updateRenderInfo(Entity entityIn, boolean p_74583_1_)
...
```

Reply:
Yes, exactly that, this change currently will break any existing mods using the method due to the way the jvm finds methods.